### ISSUE TITLE
Order of languages

### ISSUE THREAD

**Issue by matheusgomesms** [organicmaps/organicmaps.github.io#97]
In the language menu, it seems the language order is on ISO-abbreviation order, not in their names alphabetically (which I think it should be, given it is what appears to website viewers).

**Comment by biodranik** [comment]
Yes, it should be fixed. A proper place is here: https://github.com/organicmaps/organicmaps.github.io/blob/master/templates/language_selector.html
